inc/agentwakeup.class.php

```diff
@@ -48,6 +48,19 @@ class PluginGlpiinventoryAgentWakeup extends CommonDBTM
     public static $rightname = 'plugin_glpiinventory_taskjob';
 
 
+    /**
+     * Get cron task's description
+     *
+     * @return array
+     */
+    public static function cronInfo(): array
+    {
+        return [
+            'description' => __('Wake agents ups', 'glpiinventory')
```

Comment:
I have a doubt about this translation 

@cconard96 can you validate (or not)?

Reply:
A way to validate it is simply apply it, go to "Crontasks" (Automatic actions) on GLPI and see if the "Description" field is filled. This function is already used on GLPI-Inventory plug-in itself: https://github.com/glpi-project/glpi-inventory-plugin/blob/bc7d2cddeaf7641e511a5a2b3a881c54d6e5a0f4/inc/task.class.php#L756-L762

Reply:
I think he was only asking me to validate the description sounded correct in English